commands/createfixpullrequests_test.go

```diff
@@ -148,11 +153,13 @@ func TestFixPackageVersion(t *testing.T) {
 				// Fix impacted package for each technology
 				fixVersionInfo := utils.NewFixVersionInfo(test.fixVersion, test.technology, true)
 				assert.NoError(t, cfg.updatePackageToFixedVersion(test.impactedPackaged, fixVersionInfo))
-				file, err := os.ReadFile(test.packageDescriptor)
-				assert.NoError(t, err)
-				assert.Contains(t, string(file), test.fixVersion)
-				// Verify that case-sensitive packages in python are lowered
-				assert.Contains(t, string(file), strings.ToLower(test.impactedPackaged))
+				if !test.shouldNotFix {
```

Comment:
that test checks that all shouldFix deps are fixed,
but it doesn't check that all shouldnotFix deps are indeed not fixed

Reply:
done